cloudwatt/test_inter_tenant/setup_inter_tenant_3-2.yml

```diff
@@ -0,0 +1,51 @@
+- name: "Setup tools"
+  hosts: all
+  vars:
+    target_num1: 1
+    target_num2: 2
+    asn: "217.1.0.0"
+  vars_files:
+    - ansible-vars.yml
```

Comment:
The stuff about the ansible-vars.yml is still not solved. You must rely on *.tfvars file to swicth easily from one environment to another.

Think about how you would do it when you have to test quicly between lab and int

Reply:
s/*.tfvars/ansible vars file/g 

Like in the orchestration repo